Bourreau/spec/rails_helper.rb

```diff
@@ -26,7 +26,7 @@
   database      = config["database"] || "nodatabase"
   password    &&= ":#{password}"
   url           = "#{adapter}://#{username}#{password}@#{host}/#{database}"
-  ENV["DATABASE_URL"] = url
+  ENV["DATABASE_URL"] = URI.escape url
```

Comment:
This change was already in my pull request of last week... I suppose git will just merge them as-is, since they're identical.